### PR TITLE
4531 fetch data

### PR DESCRIPTION
**Pull request by TungPhanThanh**
create Track Entity, model
https://edu-redmine.framgia.vn/issues/4531

### CODE REVIEW

File: app/src/main/java/com/example/themo/musicmarvelous/constants/TrackEntity.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package com.example.themo.musicmarvelous.constants;
+
+
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/themo/musicmarvelous/constants/TrackEntity.java]: remove bl and format code

**Comment by TungPhanThanh** [comment]
ready

**Comment by NguyenMinhTuanB** [comment]
commented

**Comment by TungPhanThanh** [comment]
ready

**Pull request merged**